.github/workflows/book.yml

```diff
@@ -20,9 +20,7 @@ jobs:
           toolchain: stable
           override: true
 
-      - run: cargo build
-        working-directory: book/listings
-
+      - run: cargo build --manifest-path book/listings/Cargo.toml --bins
```

Comment:
What's the difference? working-directory does exactly what you need here.

Reply:
Good point, now that you say it :P
Fixed!